src/app/program/program-splitscreen/console/console.component.scss

```diff
@@ -21,8 +23,22 @@
     overflow-y: scroll;
     height: 0px;
     overflow-wrap: break-word;
+
+    > :first-child {
```

Comment:
otherwise the lines of output will appear at the bottom of the console, if there are not enough lines to fill the entire vertical space of the console. This way they will instead appear at the top, which is the expected behavior of a terminal.